bin/check_fixmystreet

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env perl
+
+use strict;
+use lib '/usr/lib/nagios/plugins';
+use utils qw(%ERRORS);
+use Getopt::Long;
+use FixMyStreet::DB;
+use FixMyStreet::Script::Reports;
+use Open311::PostServiceRequestUpdates;
+
+BEGIN {
+    use File::Basename qw(dirname);
+    use File::Spec;
+    my $d = dirname(File::Spec->rel2abs($0));
+    require "$d/../../fixmystreet/setenv.pl";
+}
+
+my $reports = 10;
+my $updates = 0;
+
+GetOptions(
+    "reports=i" => \$reports,
+    "updates=i" => \$updates,
+);
+
+sub unsent_report_count {
+    my $params = FixMyStreet::Script::Reports::construct_query(1);
+    my $unsent = FixMyStreet::DB->resultset('Problem')->search($params);
+
+    return $unsent->count;
+}
+
+sub unsent_update_count {
+    my $updates = Open311::PostServiceRequestUpdates->new( verbose => 1 );
```

Comment:
I don't think you want verbose (not that I think there are any log lines in fetch_bodies or construct_query, but just in case!)